README.md

```diff
@@ -23,11 +23,11 @@ enhancements are welcome.
 Running the doc server locally is easy:
 
 ```
-$ git clone git://github.com/domcleal/puppetmodule.info
-$ cd puppetmodule.info
-$ bundle install
-$ rake gems:update
-$ rackup config.ru
+git clone git://github.com/voxpupuli/puppetmodule.info
+cd puppetmodule.info
+bundle install --path .vendor --jobs $(nproc)
+rake gems:update
```

Comment:
I really wonder what this does.I really wonder what this does. I wonder if this should really be `modules:update` instead.

Reply:
indeed it's `modules:update`. Fixed it.